### PR TITLE
Update alpine to 3.9, and use separate disk for data

### PR DESCRIPTION
**Pull request by artheus**
* Create separate disk partitioned by parted for data
* Remove installation of storageos plugin, which was failing
* Use storageos server that will expose itself as a plugin

### CODE REVIEW

File: Vagrantfile [hunk 1]
```diff
@@ -17,20 +17,33 @@ vmnames = nodes.times.collect { |n| "s-#{n+1}" }
...
-  config.vm.provider "virtualbox" do |v|
-    v.memory = 2048
-    v.cpus = 2
+      node.vm.provider :virtualbox do |v|
+        v.memory = memory
+        v.cpus = cpus
+
+        #v.customize ['storagectl', :id, '--name',  'SATA Controller', '--add', 'sata',  '--controller', 'IntelAhci', '--portcount', 4]
+
+        file_to_disk = "./.vagrant/#{vmname}.vdi"
```
Comment (artheus) [inline Vagrantfile:40]: the provider part was moved into the `config.vm.define` to make use of the `vmname` variable for the extra disk filename.

File: script/run-partition-setup-and-mount [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  parted /dev/sdb mklabel gpt
+  parted /dev/sdb mkpart primary ext4 0% 100%
+
+  mkfs.ext4 -L datapartition /dev/sdb1
+fi
+
+if [ "$(cat <(mount | grep /dev/sdb1 | wc -l))" -eq '0' ]; then
+  mkdir -p /var/lib/storageos
+  mount /dev/sdb1 /var/lib/storageos
+  mount --make-shared /var/lib/storageos
```
Comment (artheus) [inline script/run-partition-setup-and-mount:20]: using `mount --make-shared` is required to make the docker volume bind-propagation `rshared` work

**Comment by artheus** [comment]
**OBS!** Seems there are some problems with the API. I get `404 Not found` for all API endpoints I test, so there is probably something wrong with the configuration.

**Comment by artheus** [comment]
As I've been going forth with this Vagrant Setup I see that Alpine (vm version) is not the os to go forth with, as the kernel is slimmed down and does not include the necessary kernel mods that StorageOS requires. I will instead setup a new project for storageos-vagrant which will use the `centos/7` box, and make use of the https://github.com/storageos/deploy alpine script for CentOS machines.

**Comment by artheus** [comment]
Here is my Vagrant project: https://github.com/artheus/storageos-vagrant
It uses the `centos/7` Vagrant box and installs using Ansible roles from https://github.com/storageos/deploy (changes has been made to ansible roles)

**Comment by croomes** [comment]
Hi @artheus, sorry you spent time on this, it's an old repo and we should have removed it.  What are you trying to do?  We have a more up-to-date Vagrant install that we use internally that we might be able to share bits of.  Feel free to msg me directly on https://slack.storageos.com/

I just saw your repo - really nice!  Please get in touch if we can help or you want to discuss.  I'm going to go ahead and un-publish this repo so we don't confuse anyone else...

**Pull request closed without merging**